Sources/Swish/Models/Scheduler.swift

```diff
@@ -1,9 +1,11 @@
+import Dispatch
+
 public typealias Scheduler = ((@escaping () -> Void) -> Void)
 
 public let immediateScheduler: Scheduler = { completion in
-  completion()
+    completion()
 }
 
 public let mainQueueScheduler: Scheduler = { completion in
-  DispatchQueue.main.async(execute: completion)
+    DispatchQueue.main.async(execute: completion)
```

Comment:
We're changing the indentation settings?

Reply:
Fixed thanks to @sharplet 's `.swiftformat` recommendation below